### PR TITLE
fix: Added web link for desktop error page

### PR DESCRIPTION
**Pull request by mananjadhav**
### Details

- Added web link for desktop error page

### Fixed Issues

$ https://github.com/Expensify/App/issues/7316

### Tests
1. Tested link for the Desktop App

- [x] Verify that no errors appear in the JS console

### QA Steps
Need to follow the reproduction steps from https://github.com/Expensify/App/pull/6856#issuecomment-1014869242 for Desktop.

- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [ ] iOS
- [ ] Android

#### Web

web-mobile-web-link

#### Desktop

desktop-link

### CODE REVIEW

**Review by parasharrajat — DISMISSED** [review]
LGTM.

cc: @NikkiWines 
:ribbon: :eyes: :ribbon:  C+ reviewed

File: src/CONST.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ const CONST = {
     // 50 megabytes in bytes
     API_MAX_ATTACHMENT_SIZE: 52428800,
     AVATAR_MAX_ATTACHMENT_SIZE: 6291456,
+    ACTIVE_ENVIRONMENT_NEW_EXPENSIFY_URL,
+
```
Comment (NikkiWines) [inline src/CONST.js:18]: extra line

**Review by NikkiWines — CHANGES_REQUESTED** [review]

**Comment by NikkiWines** [comment]
Just a note that the linked QA steps don't work currently on dev - looks like someone fixed the room details issue. I would update the QA steps to direct the tester to go to a malformed URL like `https://new.expensify.com/r/75431276____/details`

**Comment by mananjadhav** [comment]
> Just a note that the linked QA steps don't work currently on dev - looks like someone fixed the room details issue. I would update the QA steps to direct the tester to go to a malformed URL like `https://new.expensify.com/r/75431276____/details`

@NikkiWines Anyway to get this working for Desktop App? Because we can't enter URL there.

**Comment by NikkiWines** [comment]
@mananjadhav oh, right - in that case if we can use the following steps for desktop (this flow works fine on web, oddly): 

1. Create a new room using the `+` FAB 
2. Click on the room's icon to open up the room details 
3. Click on `Settings`
4. Confirm you see the error page and that it directs you to switch to web 

https://user-images.githubusercontent.com/16747822/150841469-9b20c382-2184-48f8-9e2b-90cf77b79ddb.mov

**Review by NikkiWines — APPROVED** [review]
👍

**Comment by NikkiWines** [comment]
@mananjadhav since you [mentioned tackling the url constant updates in a separate issue/PR](https://github.com/Expensify/App/issues/7316#issuecomment-1022314139), I'm going to go ahead and merge this - sound good?

**Comment by mananjadhav** [comment]
Yeah thanks, will you also help create an issue then? It'll be easier to track?

**Comment by NikkiWines** [comment]
@roryabraham created one already [here](https://github.com/Expensify/App/issues/7489)

**Pull request merged**